### ISSUE TITLE
Homepage: Black Veil Brides link not working

### ISSUE THREAD

**Issue by bogreudell** [merciba/lavarecords_lavatheme#1]

**Comment by merciba** [comment]
Background: 

I created a new entry for BVB so, not wanting to delete the original until all the settings were copied over, I renamed the old entry's slug to 'black-veil-brides-old'. When I was done, I moved it to the Trash and deleted it from the Trash as well. 

However, the new entry for BVB, despite showing the correct URL, is not linked to from the home page. This leads me to believe the old one is hanging around somewhere in the db and its slug is being printed in the homepage artist loop, located [here](https://github.com/MercibaCreative/lavarecords_lavatheme/blob/master/index.php).
